python/packages/azure-ai/agent_framework_azure_ai/_chat_client.py

```diff
@@ -1102,3 +1105,6 @@ def service_url(self) -> str:
             The service URL for the chat client, or None if not set.
         """
         return self.project_client._config.endpoint
+
+    def create_agent(self, *, store: bool | None = True, **kwargs: Any) -> "ChatAgent":
```

Comment:
I don't know if this is the way, for one, this method lacks all metadata compared to the create_agent method on the BaseClient, so it won't give the right IDE info. It also introduces a difference in creating a agent through this method, vs directly, which I don't like. I'm also not convinced what problem this solves.